doc/docusaurus/static/code/QuickStart.hs

```diff
@@ -1,26 +1,31 @@
 -- BLOCK1
 {-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE NumericUnderscores  #-}
+{-# LANGUAGE OverloadedStrings   #-}
 
 module Main where
 
 import AuctionValidator
 import Data.ByteString qualified as B
 import Data.ByteString.Base16 qualified as Base16
-import Data.ByteString.Short qualified as B
-import PlutusLedgerApi.V2 qualified as V2
+import Data.ByteString.Short qualified as BS
+import PlutusLedgerApi.V1.Crypto qualified as Crypto
+import PlutusLedgerApi.V1.Time qualified as Time
+import PlutusLedgerApi.V1.Value qualified as Value
+import PlutusLedgerApi.V3 (serialiseCompiledCode)
+import PlutusTx.Builtins.HasOpaque (stringToBuiltinByteStringHex)
 
 main :: IO ()
-main = B.writeFile "validator.uplc" . Base16.encode $ B.fromShort serialisedScript
-  where
-    script = auctionValidatorScript params
-    serialisedScript = V2.serialiseCompiledCode script
-    params =
-      AuctionParams
-        { apSeller = error "Replace with seller's wallet address"
-        , -- The asset to be auctioned is 10000 lovelaces
-          apAsset = V2.singleton V2.adaSymbol V2.adaToken 10000
-        , -- The minimum bid is 100 lovelaces
-          apMinBid = 100
-        , apEndTime = error "Replace with your desired end time"
-        }
+main =
+  B.writeFile "validator.uplc.hex"
+    $ Base16.encode
+    $ BS.fromShort
+    $ serialiseCompiledCode
+    $ auctionValidatorScript AuctionParams
+      { apSeller = Crypto.PubKeyHash (stringToBuiltinByteStringHex "")
+      , apCurrencySymbol = Value.currencySymbol "MNFT"
+      , apTokenName = Value.tokenName "NFT"
+      , apMinBid = 100
+      , apEndTime = Time.fromMilliSeconds 1_000_000 -- Replace with your desired end time
```

Comment:
If you change this, you should also update [this](https://github.com/IntersectMBO/plutus/blob/5458fe526da99a2b03f217c8fa87b1b0791fede5/doc/docusaurus/docs/auction-smart-contract/end-to-end/mint.md?plain=1#L56). But there are other `error`s, so it would make more sense to either keep all of them or change all of them.

Reply:
https://github.com/IntersectMBO/plutus-tx-template/pull/34